gbfs.md

```diff
@@ -34,12 +34,23 @@ This specification has been designed with the following concepts in mind:
 
 Historical data, including station details and ride data is to be provided by a more compact specification designed specifically for such archival purposes. The data in the specification contained in this document is intended for consumption by clients intending to provide real-time (or semi-real-time) transit advice and is designed as such.
 
+## Versioning
+The version of the GBFS documentation in which a feed is encoded is declared in all file headers. See [Output Format](#output-format).
+
+GBFS Best Practice defines that:
+* _GBFS producers_ should provide an endpoint that conforms to the current long term support (LTS) branch and the latest release branch (within 3 months of release). See [specification versioning](README.md#specification-versioning)
```

Comment:
I think this needs to be clarified a bit. Suggested text:

>_GBFS producers_ should provide endpoints that conform to both the current specification long term support (LTS) branch as well as the latest release branch.  Producers are expected to support new releases of the GBFS spec within 3 months of a new spec `MAJOR` or `MINOR` version release.

Reply:
I made the suggested changes.